pkg-config/001-pkg-config-set-glib-prefix.patch

```diff
@@ -0,0 +1,12 @@
+--- org/Makefile.vc        2017-01-23 19:27:19.473928700 +0100
++++ Makefile.vc      2017-01-23 19:25:57.354615200 +0100
+@@ -8,7 +8,9 @@
+ # in $(GLIB_PREFIX)\include\glib-2.0 and $(GLIB_PREFIX)\lib\glib-2.0\include
+ # and its import library will be found in $(GLIB_PREFIX)\lib.
+
++!if "$(GLIB_PREFIX)" == ""
```

Comment:
It doesn't seem a bug, just me not knowing how to use the program / makefile correctly :)